### PR TITLE
feat(settings): new team button on org teams page

### PR DESCRIPTION
**Pull request by MaxBittker**
screenshot 2018-02-08 17 44 16

### CODE REVIEW

**Comment by MaxBittker** [comment]
the X in the screenshot was an accident and isn't really there

File: src/sentry/static/sentry/app/views/settings/team/organizationTeamsView.jsx [hunk 1]
```diff
@@ -35,6 +37,24 @@ class OrganizationTeamsView extends React.Component {
...
+      .getAccess()
+      .has('project:admin');
+
+    let action = (
+      <Button
+        priority="primary"
+        size="small"
+        disabled={!canCreateTeams}
+        title={
+          !canCreateTeams ? t('You do not have permission to create projects') : undefined
```
Comment (lynnagara) [inline src/sentry/static/sentry/app/views/settings/team/organizationTeamsView.jsx]: should it be "teams" instead of "projects" in the message?

**Review by billyvg — APPROVED** [review]
Looks good besides the wrong message

**Comment by MaxBittker** [comment]
thanks :)

**Pull request merged**